### PR TITLE
Proposed v2.5.2 release

### PR DESCRIPTION
**Pull request by andydotxyz**
### Checklist:

- [x] Tests included.
- [x] Lint and formatter run with no errors.
- [x] Tests all pass.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,28 @@
 This file lists the main changes with each version of the Fyne toolkit.
 More detailed release notes can be found on the [releases page](https://github.com/fyne-io/fyne/releases). 
 
+## 2.5.2 - 14 October 2024
+
+### Fixed
+
+* Sometimes fyne fails to setup dark mode in Windows (#4758)
```
Comment (Jacalz) [inline CHANGELOG.md:10]: Shouldn't it be "on Windows"?

**Review by Jacalz — CHANGES_REQUESTED** [review]
Does this have the changes that I made for improving theme lookups on Linux? I couldn't see it in the changelog but it seems like something that would be useful.

**Comment by andydotxyz** [comment]
> Does this have the changes that I made for improving theme lookups on Linux? I couldn't see it in the changelog but it seems like something that would be useful.

Wasn't that resolved in 2.5.1?

>  * Light/Dark mode detection no longer works on Ubuntu with Fyne 2.5 (#5029)

File: CHANGELOG.md (hunk 1 above)
Comment (andydotxyz) [reply CHANGELOG.md]: thanks, fixed

**Comment by Jacalz** [comment]
Ah, right. Sorry 😅

**Review by Jacalz — APPROVED** [review]

**Pull request merged**